validator/src/main/resources/expected-data-template/expectedAWSSDKTrace.json

```diff
@@ -0,0 +1,59 @@
+{
+  "name": "/aws-sdk-call",
+  "fault": false,
+  "error": false,
+  "throttle": false,
+  "http": {
+    "request": {
+      "url": "http://127.0.0.1:8080/aws-sdk-call",
+      "method": "GET"
+    },
+    "response": {
+      "status": 200
+    }
+  },
+  "subsegments": [
+    {
+      "name": "DemoController.awssdkCall",
+      "fault": false,
+      "error": false,
+      "throttle": false,
+      "subsegments": [
+        {
+          "name": "S3",
+          "fault": false,
+          "error": false,
+          "throttle": false,
+          "http": {
+            "request": {
+              "url": "https://s3.us-west-2.amazonaws.com/",
```

Comment:
if we want to test in other region?

Reply:
maybe this also we can set using env variable?

Reply:
env var will work, like https://github.com/aws-observability/aws-otel-test-framework/blob/terraform/validator/src/main/java/com/amazon/aoc/validators/MetricValidator.java#L146, we can use mustachehelper to render the context object into it.

Reply:
you add some parameter into the context object

Reply:
okay will give it a shot.